packages/components/src/query-controls/index.js

```diff
@@ -12,6 +13,17 @@ import CategorySelect from './category-select';
 const DEFAULT_MIN_ITEMS = 1;
 const DEFAULT_MAX_ITEMS = 100;
 
+// currently this is needed for consistent controls UI on mobile
+// this can be removed after control components settle on consistent defaults
+const MOBILE_CONTROL_PROPS = Platform.select( {
```

Comment:
This looks familiar :grin: It's nice that the name is the same as that used in Gallery, when the defaults and fencepost logic is sorted out it'll be easier to find and refactor this pattern. :+1: